resources/css/bem/btn-osu-big.less

```diff
@@ -143,35 +143,55 @@
     --hover-bg: hsl(var(--hsl-red-2));
   }
 
+  &--download {
+    --bg:
+      var(--cookie) right 15% center no-repeat,
+      linear-gradient(var(--overlay-colour), var(--overlay-colour)) padding-box,
+      linear-gradient(
+          to bottom,
+          var(--overlay-colour),
+          hsla(var(--bg-colour), 0.8),
+          hsla(var(--bg-colour), 0.9)
+        )
+        border-box;
+
+    --hover-bg:
+      var(--cookie) right 15% center no-repeat,
+      linear-gradient(hsla(var(--bg-colour), 0.5), hsla(var(--bg-colour), 0.5))
+        padding-box,
+      linear-gradient(
+          to bottom,
+          hsla(var(--bg-colour), 0.5),
+          hsla(var(--bg-colour), 0.8),
+          hsla(var(--bg-colour), 0.9)
+        )
+        border-box;
+
+    --bg-colour: var(--hsl-pink-1);
+    --cookie: url("~@images/backgrounds/cookie_v3.svg");
+    --overlay-colour: rgba(0, 0, 0, 0.75);
+
+    .default-text-shadow();
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    border-radius: 20px;
+    border: 2px solid transparent;
+    padding: $border-radius;
+    width: 100%;
+  }
+
   &--download-landing {
     .default-box-shadow();
     .default-text-shadow();
     font-size: @font-size--title-small;
     padding: 10px 5px;
   }
 
-  &--download {
-    .default-box-shadow();
-    .default-border-radius();
-    .center-content();
-    transition: background-position 120ms;
-
-    flex: none;
-    min-width: 240px;
-    padding: 10px;
-    margin-bottom: 5px;
-    line-height: 1;
-    gap: 5px;
-
-    flex-direction: column;
-    --bg-colour: hsl(var(--hsl-blue-3));
-    --bg: var(--bg-colour) center/150% url("~@images/backgrounds/button.svg");
-    --hover-bg: var(--bg-colour) calc(50% - 20px) 50%/150%
-      url("~@images/backgrounds/button.svg");
-  }
-
-  &--download-lazer {
-    --bg-colour: hsl(var(--hsl-purple-3));
+  &--download-stable {
+    --bg-colour: var(--hsl-yellow-1);
```

Comment:
where did this come from? shouldn't it be orange? (and where did the yellow definition come from?)